cmds/core/date/date_linux.go

```diff
@@ -0,0 +1,22 @@
+// Copyright 2015-2017 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// +build !plan9
```

Comment:
remove this, it's implied from the filename

Reply:
I forgot to change that to date_unix.go, oos.

Reply:
name changed. It builds for freebsd too, so the _linux.go made no sense.